packages/language-server/src/completion/completions.ts

```diff
@@ -716,14 +716,30 @@ function getSuggestionsForAttribute(
     if (attribute && attribute !== '@@fulltext' && isInsideAttribute(untrimmedCurrentLine, position, '[]')) {
       if (isInsideFieldArgument(untrimmedCurrentLine, position)) {
         // extendedIndexes
-        return {
-          items: filterSortLengthBasedOnInput(
+        const items: CompletionItem[] = []
+        // https://github.com/prisma/language-tools/issues/1139
+        // https://www.notion.so/prismaio/Proposal-More-PostgreSQL-index-types-GiST-GIN-SP-GiST-and-BRIN-e27ef762ee4846a9a282eec1a5129270
```

Comment:
I'd keep the notion link at hand, but I think we can get rid of the issue link (if one needs it, they can retrieve it from `git blame`)

```suggestion
        // https://www.notion.so/prismaio/Proposal-More-PostgreSQL-index-types-GiST-GIN-SP-GiST-and-BRIN-e27ef762ee4846a9a282eec1a5129270
```